### ISSUE TITLE
Lay out Completed segment

### ISSUE THREAD

**Issue by kmeyersvt** [codeforbtv/cvoeo-app#35]
Goal: as a user, I want to see what I've completed already
Why?: so the user can see the work that they've already done successfully
DoD: 
- If there are no completed goals yet, display a message like "Keep up the good work, you'll finish a goal soon!"
- If there is one goal completed, display the goal.
- If there is more than one goal completed, display the first one in the quadrant and display the rest when the expansion arrow is clicked
- There is a visual indicator that there are more completed items by changing how the expansion arrow looks

**Comment by kmeyersvt** [comment]
Unblocking because the design has been approved.

**Comment by doub1ejack** [comment]
@jbpayne I think you've done this already.  Can you confirm & close this issue?
